railties/test/application/configuration_test.rb

```diff
@@ -2226,6 +2226,36 @@ class ::DummySerializer < ActiveJob::Serializers::ObjectSerializer; end
       assert_equal true, ActiveJob::Base.return_false_on_aborted_enqueue
     end
 
+    test "ActiveStorage.queues[:analysis] is :active_storage_analysis by default" do
+      app "development"
+
+      assert_equal :active_storage_analysis, ActiveStorage.queues[:analysis]
+    end
+
+    test "ActiveStorage.queues[:analysis] is nil by default for upgraded apps" do
+      remove_from_config '.*config\.load_defaults.*\n'
+      add_to_config 'config.load_defaults "5.2"'
+
+      app "development"
+
+      assert_nil ActiveStorage.queues[:analysis]
+    end
+
+    test "ActiveStorage.queues[:purge] is :active_storage_purge by default" do
+      app "development"
+
+      assert_equal :active_storage_purge, ActiveStorage.queues[:purge]
+    end
+
+    test "ActiveStorage.queues[:purge] is nil by default for upgraded apps" do
```

Comment:
The names of those test would be invalid when Rails 6.0 is out maybe it would be better to say "... is nil without the default configurations"